src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/ExecutableStatementCountCheck.java

```diff
@@ -180,7 +186,8 @@ public void leaveToken(DetailAST ast) {
             case TokenTypes.INSTANCE_INIT:
             case TokenTypes.STATIC_INIT:
             case TokenTypes.COMPACT_CTOR_DEF:
-                leaveMemberDef(ast);
+            case TokenTypes.LAMBDA:
```

Comment:
lets reuse `isContainerNode` method here, like:
```
if (isContainerNode(ast)) {
   leaveContainerNode(ast);
} else 
if (TokenTypes.SLIST == ast.getType()) {
// Do nothing
} else {
   throw new IllegalStateException(ast.toString());
}
```

Reply:
same idea for `visitToken`

Reply:
done